sample-plugin-bar-chart/README.md

```diff
@@ -0,0 +1,65 @@
+# Getting Started with Sigma Plugins
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+&nbsp;
+
+## Running This React App
+
+If this is your first time running this app, navigate to the project directory and run the following command via your terminal:
+
+### `yarn`
+
+- Installs all necessary dependencies.
+
+- You may skip this step if you have already run this command for this app previously.
+
+After this, while still in the project directory, you can run:
+
+### `yarn start`
+
+- Runs the app in the development mode. Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+- The page will reload if you make edits. You will also see any lint errors in the console.
+
+&nbsp;
+
+## Using This App as a Sigma Plugin
+
+Start off by creating a new workbook or opening an existing workbook. Make sure that your workbook has a data table in it that the plugin can read from.
+
+Open the "Add element..." tab in the sidebar and add a Plugin (under "UI Elements").
+
+<img src="readme_images/add-element.png" alt="screenshot of 'Add element...' tab on the top left section of the workbook view" width="300">
+<img src="readme_images/plugin.png" alt="screenshot of Plugin element under 'UI Elements' section" height="400" align="top">
+
+Select the "Sigma Plugin Dev Playground" plugin, which plugin developers should have access to. Please contact your Organization Administrator if you cannot find this.
```

Comment:
@xyin96 double checking here - this is just something that we added to SoS and orgs will have to add an equivalent version themselves, right?

Reply:
it's not automatically available for every org by default. Maybe we can add a section on how to register this plugin with the org. (see: https://docs.google.com/document/d/1PgYD3mTfLUQSHRbiccU7glER-v41YInNMYcDdzca-iM/edit#heading=h.d8bv4lryinjp)